src/appkit.cs

```diff
@@ -2619,7 +2619,7 @@ public interface NSCollectionView : NSDraggingSource, NSDraggingDestination {
 		//NSImage DraggingImage (NSIndexSet itemIndexes, NSEvent evt, NSPointPointer dragImageOffset);
 
 		//Detected properties
-		[Export ("delegate", ArgumentSemantic.Assign), NullAllowed]
+		[Export ("delegate", ArgumentSemantic.Weak), NullAllowed]
```

Comment:
Changing `Assign` to `Weak` should always be okay since they "almost" mean the same thing. IIRC weak properties are set to nil when deallocated in ARC environments instead of pointing to garbage when `assign` is used.